static/js/portico/team.js

```diff
@@ -92,8 +98,11 @@ export default function render_tabs() {
 
         $(`#${CSS.escape(tab_name)}`).on("click", () => {
             if (!loaded_repos.includes(repo_name)) {
-                const filtered_by_repo = contributors_list.filter((c) => c[repo_name]);
-                const html = filtered_by_repo
+                const filtered = contributors_list
+                    .filter((c) => c[repo_name])
+                    // Exclude entries by dependabot
+                    .filter((c) => exclude_github_username(c, "dependabot[bot]"));
```

Comment:
It's not great that we need to do this filtering in two different places, since that's the kind of pattern likely to lead to bugs in the future -- ideally we'd do it in a single place when sourcing the data, either in the tool that fetches data from GitHub or in a preprocessing step.